llmfoundry/callbacks/__init__.py

```diff
@@ -18,12 +20,7 @@
     ) from e
 
 __all__ = [
-    'FDiffMetrics',
-    'MonolithicCheckpointSaver',
-    'GlobalLRScaling',
-    'LayerFreezing',
-    'ScheduledGarbageCollector',
-    'EvalGauntlet',
-    'HuggingFaceCheckpointer',
-    'AsyncEval',
+    'FDiffMetrics', 'MonolithicCheckpointSaver', 'GlobalLRScaling',
+    'LayerFreezing', 'ScheduledGarbageCollector', 'EvalGauntlet',
+    'HuggingFaceCheckpointer', 'AsyncEval', 'CurriculumLearning'
```

Comment:
```suggestion
    'HuggingFaceCheckpointer', 'AsyncEval', 'CurriculumLearning',
```
nit restore trailing comma

Reply:
updated, linting changed formatting